src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -369,6 +369,8 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       batchReindex: `${KIBANA_DOCS}batch-start-resume-reindex.html`,
       remoteReindex: `${ELASTICSEARCH_DOCS}docs-reindex.html#reindex-from-remote`,
       reindexWithPipeline: `${ELASTICSEARCH_DOCS}docs-reindex.html#reindex-with-an-ingest-pipeline`,
+      logsDatastream: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/current/logs-data-stream.html`,
+      usingLogsDbIndexModeWithESSecurity: `${ELASTIC_WEBSITE_URL}guide/en/security/current/detections-logsdb-index-mode-impact.html`,
```

Comment:
Should this link to a specific docs version instead of `current` since the callout is only going into 8.x?

Reply:
Good point, likely need to inject the version value there.